devices.py

```diff
@@ -59,35 +61,296 @@ class Newport(EpicsMotor):
     """
     Basic class for the newport motors.
     """
-    home_forward = C(Signal)
-    home_reverse = C(Signal)
-    offset_freeze_switch = C(Signal)
+
+    home_forward = Cpt(Signal)
+    home_reverse = Cpt(Signal)
+    offset_freeze_switch = Cpt(Signal)
 
     #@retry(tries=3)
     def move(self, *args, **kwargs):
         return super().move(*args, **kwargs)
 
 
-class SeqBase(SndDevice):
+class Sequencer(Device):
     """
     Base sequencer class.
+    
+    Attributes
+    ----------
+    state_control : Component
```

Comment:
While these signals are being added as `Component`, during the initialization of the object, they get changed into whichever class is passed in the `Component` call. In this case, `state_control` will be of type `EpicsSignal`.